linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_warn_age_existing/rule.yml

```diff
@@ -0,0 +1,47 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: "Set Existing Passwords Warning Age"
+
+description: |-
+    Configure non-compliant accounts, and to give users time of think of a secure password,
+    to receive number of days or more, according to the days specified with the variable   
+    var_accounts_password_warn_age_login_defs, an advance warning that a password will 
+    be expiring by running the following command:
+    <pre>$ sudo chage --warndays $var_accounts_password_warn_age_login_defs <i>USER</i></pre>
+
+rationale: |-
+    Providing an advance warning that a password will be expiring gives users 
+    time to think of a secure password. Users caught unaware may choose a simple 
+    password or write it down where it may be discovered.
+
+severity: medium
+
+identifiers:
+    cce@sle12: CCE-92321-9
+    cce@sle15: CCE-92479-5
+
+references:
+    cis@sle12: 5.4.1.4
+    cis@sle15: 5.4.1.4
+    disa: CCI-000198
+    nist: IA-5(f),IA-5(1)(d),CM-6(a)
+    nist@sle15: IA-5(1).1(v)
+
+ocil_clause: 'any results are returned that are not associated with a system account'
+
+ocil: |-
+    Verify that {{{ full_name }}} has configured the warning that a password will be expiring for each user account 
+    is number of days or greater, according to the days specified with the variable 
+    var_accounts_password_warn_age_login_defs,  with the following command:
```

Comment:
Also here, I believe the previous text was better. It was only necessary to change `7` by `{{{ xccdf_value("var_accounts_password_warn_age_login_defs") }}}`

Reply:
Here was not updated in the last commit.

Reply:
@rumch-se , did you note this comment from my first review? We already know the value of `var_accounts_password_warn_age_login_defs` at this point. So, we can inform the value itself instead of mentioning the variable name literally. If the user is reading this, he has to stop the flow and go to another place to consult the variable value. I believe this experience is improved by promptly providing the necessary information. This is also valid for the description.